content/intro-to-storybook/angular/es/simple-component.md

```diff
@@ -1,378 +1,381 @@
 
-```typescript
-import { configure } from '@storybook/angular';
+`action()` nos permite crear una devolución de la llamada que aparece en el panel **actions** de la UI de Storybook al hacer clic. Entonces, cuando creamos un botón de marcador, podremos determinar en la UI de prueba si el clic de un botón es exitoso.
 
-import '../src/styles.less';
+Como necesitamos pasar el mismo conjunto de acciones a todas las permutaciones de nuestro componente, es conveniente agruparlas en una sola variable `actionsData` y pasarlas a nuestra definición de historia cada vez.
 
-// automatically import all files ending in *.stories.ts
-const req = require.context('../src/', true, /\.stories.ts$/);
+Otra cosa buena de agrupar los `actionsData` que necesita un componente, es que puedes` exportarlos` y usarlos en historias para componentes que reutilizan este componente, como veremos más adelante.
 
-function loadStories() {
-  req.keys().forEach(filename => req(filename));
-}
+<div class="aside">
+💡 <a href="https://storybook.js.org/docs/angular/essentials/actions"><b>Actions</b></a> nos ayuda a verificar las 
+interacciones al crear componentes de UI de forma aislada. A menudo, no tendremos acceso a las funciones y el estado 
+que tiene el contexto de la aplicación. Use <code>action()</code> para simularlas.
+</div>
 
-configure(loadStories, module);
-```
+## Configuración
 
-Para que sea posible importar directamente nuestro archivo LESS necesitamos añadir una configuración expecial de Webpack. Basta con crear un archivo `webpack.config.js` dentro del directorio `.storybook` y pegar el siguiente código:
+Tendremos que hacer un par de cambios en la configuración de Storybook para que note nuestras historias creadas recientemente y nos permita mostrar correctamente las historias de nuestros componentes.
 
-```javascript
-const path = require('path');
+Comience cambiando su archivo de configuración de Storybook (`.storybook/main.js`) a lo siguiente:
 
+```diff:title=.storybook/main.js
 module.exports = {
-  module: {
-    rules: [
-      {
-        test: /\.less$/,
-        loaders: ['style-loader', 'css-loader', 'less-loader'],
-        include: path.resolve(__dirname, '../'),
-      },
-    ],
-  },
+- stories: [
+-   '../src/**/*.stories.mdx',
+-   '../src/**/*.stories.@(js|jsx|ts|tsx)'
+- ],
++ stories: ['../src/app/components/**/*.stories.ts'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
 };
 ```
 
-También es necesario instalar los siguiente paquetes:
+Después de completar el cambio anterior, dentro de la carpeta `.storybook`, cambie se `preview.js` a lo siguiente:
 
+```diff:title=.storybook/preview.js
+import { setCompodocJson } from "@storybook/addon-docs/angular";
+import docJson from "../documentation.json";
+setCompodocJson(docJson);
+
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
++ angularLegacyRendering: true,
+}
```

Comment:
@josegarciamanez if you could adjust this portion to what's currently in the English version we'd appreciate it. I pushed a small pull request #488 that addressed this. Long story short this was introduced as a temporary solution that I was able to sort out with the current Storybook Angular maintainer and with his guidance I updated the tutorial accordingly.